modules/custom-status/custom-status.php

```diff
@@ -162,6 +162,14 @@ function install() {
 					'position' => 5,
 				),
 			),
+			array(
+				'term' => __( 'Expired', 'edit-flow' ),
+				'args' => array(
+					'slug' => 'expired',
+					'description' => __( 'Post has expired, and it has to bew updated an editor.', 'edit-flow' ),
```

Comment:
Think there's a typo. Here's a possible condensed version:

```suggestion
					'description' => __( 'Post has expired and requires updates .', 'edit-flow' ),
```